ironfish/src/network/peers/peer.ts

```diff
@@ -190,6 +192,16 @@ export class Peer {
 
   loggedMessages: Array<LoggedMessage> = []
 
+  /**
+   * Blocks that have been sent or received from this peer. Value is set to true if the block was received
+   * from the peer, and false if the block was sent to the peer.
+   */
+  readonly knownBlockHashes: LRU<Buffer, boolean> = new LRU<Buffer, boolean>(
```

Comment:
There is an alias type we commonly use around if you want to use it, or not. `BlockHash`